### ISSUE TITLE
Default avatar instead of photo is shown after making a photo in Profile [bug/username-changed-to-default]

### ISSUE THREAD

**Issue by annadanchenko** [status-im/status-react#1769]
### Description

*Type*: Bug

*Summary*: Default avatar instead of a photo is shown after making a photo in Profile
![screenshot_20170906-162800](https://user-images.githubusercontent.com/6584810/30114528-c2e4c9ba-9320-11e7-8edd-412a570440d4.png)

#### Expected behavior
After making a photo it should be shown on Edit Profile screen
#### Actual behavior
 Default avatar instead of a photo is shown after making a photo

### Reproduction
From 00:19 https://app.testfairy.com/projects/4803622-status/builds/6498698/sessions/14/?accessToken=NozatymvQ3sBc/avCvvOGw9aKj4

- Open Status
- Open drawer
- Open Profile screen (tap on 3 dots icon)
- Edit Profile (tap on 3 dots icon and select Edit)
- On avatar picture inside circle tap on Edit
- Tap on "Capture" (if it's 1st time then allow access in popup)
- On Profile image screen tap on "camera" icon to make a selfie. Note: try to tap only once and wait, do not tap several times as there is another issue with camera not taking pictures.
- After you tap on camera app should return to the screen Edit Profile and inside circle for avatar you should see taken picture. Currently, issue is that app returns to the expected screen but no photo is shown in the circle for avatar. There is same old default avatar shown.

### Additional Information
* Status version: PR `bug/username-changed-to-default` 0.9.10-61-gdeb1dbbb+
* Operating System: 
Real device Samsung Galaxy S6, Android 6.0.1
Real device iPhone 6s, iOS 10.3.3
